screens/ProfilePage.tsx

```diff
@@ -1,55 +1,104 @@
 import * as React from "react";
-import {
-  StyleSheet,
-  Text,
-  View,
-  TextInput,
-  TouchableOpacity,
-} from "react-native";
-
-interface UserInfo {
-  email: string;
-  username: string;
-  fullName: string;
-  phone: string;
-}
+import { useState } from "react";
+import { userContext } from "../App";
+//import the context
+import { useUserContext } from "../App";
+import { StyleSheet, Text, View, Pressable, Alert } from "react-native";
 
 const ProfilePage = () => {
-  const [userInfo, setUserInfo] = React.useState<UserInfo>({
-    email: "",
-    username: "",
-    fullName: "",
-    phone: "",
-  });
+  //extract user info from userContext (aka the user information)
+  const { info } = useUserContext();
+  const onSelling = () => {
+    Alert.alert("Switch to selling");
+    setShowingSelling(true);
+  };
+  const onWishlist = () => {
+    Alert.alert("Switch to wishlist");
+    setShowingSelling(false);
+  };
+  const [showingSelling, setShowingSelling] = useState(true);
 
   return (
-    <View style={styles.container}>
-      <Text style={styles.header}>Name: </Text>
-      <Text style={styles.info}>{userInfo.fullName}</Text>
-      <Text style={styles.header}>Username: </Text>
-      <Text style={styles.info}>{userInfo.username}</Text>
-      <Text style={styles.header}>Email: </Text>
-      <Text style={styles.info}>{userInfo.email}</Text>
-      <Text style={styles.header}>Phone: </Text>
-      <Text style={styles.info}>{userInfo.phone}</Text>
+    <View style={styles.page}>
+      <View style={styles.profileInfo}>
+        <Text style={styles.fullName}>{info.fullName}</Text>
+        <Text style={styles.username}>@{info.username}</Text>
+        <Text style={styles.phoneNumber}>&#128222;{info.phoneNumber}</Text>
+        <View style={styles.fixToText}>
+          <Pressable style={styles.sellingButton} onPress={onSelling}>
+            <Text style={styles.text}>Selling</Text>
+          </Pressable>
```

Comment:
Just something to keep in mind that we probably want the pressables "Selling" and "Wishlist" to change color based on which one is being displayed (the darker one being the one that is displayed etc) but not necessarily a problem for now.